integrationtests/src/test/java/io/kroxylicious/proxy/filter/ApiVersionsMarkingFilter.java

```diff
@@ -39,4 +41,21 @@ public CompletionStage<RequestFilterResult> onRequest(ApiKeys apiKey, RequestHea
             return context.forwardRequest(header, request);
         });
     }
+
+    public static class Contributor implements FilterContributor {
+        @Override
+        public String getTypeName() {
+            return "ApiVersionsMarkingFilter";
+        }
+
+        @Override
+        public ConfigurationDefinition getConfigDefinition() {
+            return new ConfigurationDefinition(BaseConfig.class, false);
```

Comment:
could we simplify the coding required for a filter without config? wonder if Contributor should have a default getConfigDefinition method  that returns the no configuration  `new ConfigurationDefinition(BaseConfig.class, false)`

Reply:
added default